xbmc/pvr/channels/PVRChannelGroups.cpp

```diff
@@ -237,6 +237,9 @@ bool CPVRChannelGroups::Update(bool bChannelsOnly /* = false */)
       RemoveFromAllGroups(channelsToRemove);
     }
 
+    if (bReturn && group == m_selectedGroup)
```

Comment:
Why is it enough to update only the selected group? What happens when the user selects another group? Will the data for that group be up-to-date, then?

Reply:
Because the update will already happen as part of `SetSelectedGroup` each time the user selects a different group.

So we just need to update the selected group in case there are new or changed channels.

Reply:
okay